crt_portal/cts_forms/templates/landing.html

```diff
@@ -275,7 +275,7 @@ <h2 class="h1__display text__reverse" id="report-a-violation">{% trans "How to r
         </div>
         <div class="tablet:grid-col-12">
           <div class="grid-row grid-gap">
-            <div class="tablet:grid-col-4">
+            <div class="tablet:grid-col-4 crt-landing--section__item">
               <div class="crt-landing--reporting_column">
                 <div class="h4__display crt-landing--icon_gold">1</div>
                 <div>
```

Comment:
👇 on line 282, 291, and 300, we need to add the class `crt-landing--subheader` to the `h2` elements in order to avoid extra spacing showing up above the gold bullet points:
<img width="1024" alt="Screen Shot 2020-07-28 at 2 17 29 PM" src="https://user-images.githubusercontent.com/509309/88723494-35175b00-d0e6-11ea-91ec-99628a0a1ee4.png">